vm/Cargo.toml

```diff
@@ -19,6 +19,7 @@ std = [
     "parse-hyperlinks/std",
     "felt/std",
     "dep:num-prime",
+    "thiserror-no-std/std",
```

Comment:
All of this makes me wonder, do we actually use anything that requires the `std` feature on `thiserror`? Did you try building without it?

Reply:
In the future rust will move the error in `core`